lib/locales/en.ts

```diff
@@ -507,6 +520,8 @@ export const locale: Locale = {
     innocent_v_pope: 'Blessed Innocent V, Pope',
     innocent_xi_pope: 'Blessed Innocent XI, Pope',
     irenaeus_of_lyon_bishop: 'Saint Irenaeus, Bishop, Martyr and Doctor of the Church',
+    irenaeus_of_lyon_bishop_patron_of_the_archdiocese_of_lyon_and_companions_martyrs:
+      'Saints Irenaeus, Bishop and Doctor of the Church, Principal Patron of the Archdiocese of Lyon, and Companions, Martyrs',
```

Comment:
In `en.ts`, we don’t use `Principal Patron`, but simply `Patron`. Secondary patrons are `copatrons`.

```suggestion
      'Saints Irenaeus, Bishop and Doctor of the Church, Patron of the Archdiocese of Lyon, and Companions, Martyrs',
```

Reply:
Got it 👍